Makefile

```diff
@@ -74,11 +70,11 @@ test: ## Run the go tests ($ go test -v ./cmd/... ./pkg/...)
 
 .PHONY: test-ci
 test-ci: ## Run the unit tests (used in the CI)
-	./setup.sh
 	./test.sh
 
-.PHONY: test-e2e
+.PHONY: test-ci-e2e
```

Comment:
Why are you changing the name? Where are this tests run should not be part of their name.

Reply:
We have `test-ci` for the unit tests then, make sense keep the same approach to make clear its usage.

Reply:
The target is `test-e2e, having:
```
.PHONY: test-ci-e2e
test-e2e: ...
```
makes no sense.

Reply:
Oh. Now I got what do you mean. Solved 👍

Reply:
You changed both to `test-ci-e2e` and that wasn't exactly what I meant.

You should probably also have to look in some `.md` files for instructions that used to call `make test-e2e`.